.github/workflows/ci.yaml

```diff
@@ -274,7 +289,7 @@ jobs:
 
       - name: Change name of deployment file for staging deployment
         if:
-          github.event_name == 'pull_request' && matrix.environment == 'staging'
+          matrix.environment == 'staging'
```

Comment:
The idea is to also push to `staging_candidates/` a release file when the push to master happens. This is so we can more easily create the deployment from the script when needed.